pycodestyle.py

```diff
@@ -328,12 +335,17 @@ def blank_lines(logical_line, blank_lines, indent_level, line_number,
     E305: def a():\n    pass\na()
     E306: def a():\n    def b():\n        pass\n    def c():\n        pass
     """
+    top_level_lines = BLANK_LINES_CONFIG['top_level']
+    method_lines = BLANK_LINES_CONFIG['method']
```

Comment:
Thanks for the review. Will update it.

I will look at updating all the code. I was not sure this will be accepted and it was a quick hack to demonstrate the direction.

--------

I was expecting that for each instance where pycodestyle asks for 2 blank line, for Twisted we will have 3 :)..and for all other insances where pycodestyle asks for 1 blank line for Twisted there will be 2... reusing the same code number.

This is how it is used now... but with a hardcore fork which is a bit hard to maintain :) https://github.com/twisted/twistedchecker/blob/master/twistedchecker/checkers/pycodestyleformat.py#L218

----------

I was thinking to just disable E3XX from pycodestyle and register a new checker with custom code but this would lead to a lot of code duplication.

This is why I am proposing this path :)

Reply:
I totally understand the _why_ behind this, and agree with it. I wonder if it would be plausible to have some tests which modify these values and assert compliant code, outside of the testsuite.

Reply:
Yes. I want to write tests... no doubt about it... is just that I am confused where and how I am expected to write the tests... see testsuite/E30not.py  which fails on master.

I was thinking to create a new testsuite/test_blank_lines.py file where to write some integration tests for the blank lines code.

I am open to any other suggestion.